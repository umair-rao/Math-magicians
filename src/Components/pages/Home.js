import React from 'react';

export default function Home() {
  return (
    <div className="homepage">
      <h1>Welcome to our page</h1>
      <p className="homepage-detail">
        If we learn mathematics, we can gain several very beneficial
        uses for our economic growth and thinking. It improves
        our critical thinking, aids logical thinking, improves our minds,
        encourages practicality, and could be used in daily situations.
      </p>
    </div>
  );
}
