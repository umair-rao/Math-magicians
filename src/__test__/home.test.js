import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Components/pages/Home';

it('renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
