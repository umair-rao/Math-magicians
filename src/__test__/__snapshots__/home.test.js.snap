// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="homepage"
>
  <h1>
    Welcome to Math Magician
  </h1>
  <p
    className="homepage-detail"
  >
    If we learn mathematics, we can gain several very beneficial uses for our economic growth and thinking. It improves our critical thinking, aids logical thinking, improves our minds, encourages practicality, and could be used in daily situations.
  </p>
</div>
`;
